e2e/cypress/integration/notifications/browser_tab_notification_spec.js

```diff
@@ -96,8 +98,8 @@ describe('Notifications', () => {
             cy.get('.badge').contains('1');
         });
 
-        // * Favicon should stay blue (not turn red) when there are mentions indicated
-        verifyFaviconEquals('favicon-default-16x16.png');
+        // * Favicon should turn red when there are mentions indicated
+        verifyFaviconEquals('favicon-mentions-16x16.png');
```

Comment:
@saturninoabril I moved `Remove mention notification (for initial channel).` to run on before()(ln45), then It worked that we discussed in https://github.com/mattermost/mattermost-server/issues/15703.